src/marketplace/marketplace.service.spec.ts

```
@@ -0,0 +1,206 @@
+import {
+  passportTemplateDtoFactory,
+  templateDataFactory,
+} from './fixtures/passport.template.factory';
+import { MongooseTestingModule } from '../../test/mongo.testing.module';
+import { MongooseModule } from '@nestjs/mongoose';
+import { laptopFactory } from '../templates/fixtures/laptop.factory';
+import { TemplateService } from '../templates/infrastructure/template.service';
+import { templateCreatePropsFactory } from '../templates/fixtures/template.factory';
+
+export const mockCreatePassportTemplateInMarketplace = jest.fn();
+export const mockGetPassportTemplateInMarketplace = jest.fn();
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)